sdk/iOS/src/WindowsAzureMobileServices.h

```diff
@@ -11,7 +11,7 @@
 #import "MSError.h"
 #import "MSFilter.h"
 #if TARGET_OS_IPHONE
-#import "MSLoginController.h"
+#import <WindowsAzureMobileServices/MSLoginController.h>
```

Comment:
Not sure to be honest. 

Although if you look at the Apple umbrella headers for their frameworks they list it with the angle brackets and full path e.g. inside `#import <Foundation/Foundation.h>` they have `#import <Foundation/NSArray.h>` so it's probably just a convention for header searching.

I can change it to just the file to be consistent with the others